src/main/java/org/checkerframework/checker/dependencyinjection/qual/Bind.java

```diff
@@ -8,22 +8,22 @@
 import org.checkerframework.framework.qual.SubtypeOf;
 
 /**
- * If an expression has type {@code @Bind({"foo"})}, then {@code foo.class} has been passed as an
- * argument to {@code bind} in a Guice module.
+ * Represents classes and names that have been passed as arguments to {@link
+ * com.google.inject.Binder#bind()}.
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
 @SubtypeOf({})
 @DefaultQualifierInHierarchy
 public @interface Bind {
+  /** The fully-qualified name of this annotation. */
+  static final String NAME = "org.checkerframework.checker.dependencyinjection.qual.Bind";
```

Comment:
This is a good idea, and we should probably do this is a matter of course for all new annotations. I like it :)